.github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md

```diff
@@ -0,0 +1,48 @@
+---
+name: 'Runtime Upgrade: [Upgrade Title/Version]'
+about: Checklist for tracking the progress of a runtime upgrade on the Paseo Testnet.
+title: ''
+labels: runtime-upgrade
+assignees: ''
+
+---
+
+# Paseo Testnet Runtime Upgrade Checklist
+
+## Preparation Tasks
```

Comment:
Should we include an extra DB backup before enacting any upgrade ?

As an effort of having the means to a smooth recovery in the scenario where something fails for whatever reason. Avoiding network resets as much as we possibly can .

Reply:
i cannot agree more.